### PR TITLE
Stream with nemesis

### PR DESCRIPTION
**Pull request by yarongilor**
## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Review by fruch — APPROVED** [review]
LGTM, we'll merge, after review the results of the run

File: sdcm/kcl_thread.py [hunk 1]
```diff
@@ -125,20 +125,19 @@ def _run_stress(self, loader, loader_idx, cpu_idx):
...
+                src_size = node.get_cfstats(src_table)['Number of partitions (estimate)']
 
+                node.running_nemesis = None
                 status = f"== CompareTablesSizesThread: dst table/src table number of partitions: {dst_size}/{src_size} =="
                 LOGGER.info(status)
                 InfoEvent(status)
 
                 if src_size == 0:
                     continue
-                if dst_size >= src_size:
```
Comment (fruch) [inline sdcm/kcl_thread.py:139]: without some kind of stopping logic, this would run forever

Comment (yarongilor) [reply sdcm/kcl_thread.py:139]: added timeout to break out of loop.

**Review by fruch — APPROVED** [review]
LGTM

**Review by fruch — APPROVED** [review]
LGTM

**Pull request merged**